### PR TITLE
converted from tachyons to tailwind

### PR DESCRIPTION
**Pull request by zjael**
Fixes issue #286

Changes: 
Converted from tachyons to tailwind - as close as i could
Added npm script for building tailwind css file

Screenshots for the change:
Original:
original
Tailwind version:
tailwind

### CODE REVIEW

**Comment by zjael** [comment]
Alright done, lets me know if there is anything else. 😊

**Comment by jenkoian** [comment]
Just out of interest have you looked into any of the [custom css](https://github.com/jenkoian/hacktoberfest-checker/blob/master/public/css/main.css) and seeing if we can just do it with utility classes from tailwind?

**Comment by zjael** [comment]
Hmm, the colors can be removed and added to the tailwind.js file instead, can get rid of "main-content, fb-share-button" too, but don't think there is much besides that. 🤔

**Comment by jenkoian** [comment]
@zjael sorry pal, not had time to give this a proper review, looks brilliant, just wanna check it out and have a play around, hopefully soon 👍

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "start": "node index.js",
     "babel-dev": "babel public/js/*.js --presets es2015 --out-file public/js/main-compiled.js --source-maps --ignore *-compiled.js --watch",
     "dev": "nodemon index.js",
+    "tailwind-gen": "./node_modules/.bin/tailwind.cmd build ./tailwind-gen.css -c ./tailwind.js -o ./public/css/tailwind.css",
```
Comment (jenkoian) [inline package.json]: Guessing you're on windows, `tailwind.cmd` doesn't exist on *nix systems it seems, so wondering if we can update this to simply use `./node_modules/.bin/tailwind`

**Comment by jenkoian** [comment]
Sorry I know this is a bit picky but on mobile, the text in the top banner goes under the github corner thing, but doesn't seem to with tachyons, I've had a quick look but nothing obvious comes to mind, are you able to take a little look at this? Also, I've merged some other PRs so you'll like need to rebase and fix the conflicts.

Overall, you've done an awesome job with this mate 👍

**Comment by zjael** [comment]
Alright, should be good to go now, let me know if there is anything else. 😊

**Comment by jenkoian** [comment]
Hey, really sorry, just merged another long running PR which means you'll need to rebase and resolve conflicts, if you're able to do that, I'll give it a check and hopefully merge, the work you've done here is tops :)

**Comment by zjael** [comment]
Alright done. 😃

**Comment by jenkoian** [comment]
Hey just checked this and all looks good, just one minor thing. The styling on the facebook share button seems to have disappeared?

**Comment by jenkoian** [comment]
Thanks for your patience!

**Pull request merged**